README.md

```diff
@@ -0,0 +1,73 @@
+# Theia - SSH Extension
+The example of how to build the Theia-based applications with the SSH extension.
+
+## Getting started
+
+Install [nvm](https://github.com/creationix/nvm#install-script).
+
+    curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.5/install.sh | bash
+
+Install npm and node.
+
+    nvm install 8
+    nvm use 8
+
+Install yarn.
+
+    npm install -g yarn
+
+## Running the browser example
+
+    yarn rebuild:browser
+    cd browser-app
+    yarn start
+
+Open http://localhost:3000 in the browser.
+
+## Running the Electron example
+
+    yarn rebuild:electron
+    cd electron-app
+    yarn start
+
+## Developing with the browser example
+
+Start watching of the hello world extension.
+
+    cd ssh-extension
+    yarn watch
+
+Start watching of the browser example.
+
+    yarn rebuild:browser
+    cd browser-app
+    yarn watch
+
+Launch `Start Browser Backend` configuration from VS code.
+
+Open http://localhost:3000 in the browser.
+
+## Developing with the Electron example
+
+Start watching of the hello world extension.
```

Comment:
it's not hello world so maybe readme needs to be updated/reduced

Reply:
I've updated README.md and contributed [a related fix](https://github.com/theia-ide/generator-theia-extension/pull/15) to Theia extension generator. Thanks for noticing!